index.html

```diff
@@ -19,27 +19,32 @@
 <div class="rel_boxtext">
 
 <p>Currently, <a href="//llvm.org">LLVM</a> releases are distributed under the
-<a href="7.0.0/LICENSE.TXT">University of Illinois Open Source License</a>, an
-<a href="http://www.opensource.org/licenses/UoI-NCSA.php">OSI-approved</a>
+<a href="https://llvm.org/docs/DeveloperPolicy.html#new-llvm-project-license-framework">
+Apache License, Version 2.0 with exceptions</a>, an
```

Comment:
The probably most technically correct way to refer to the license is using [SPDX](https://spdx.dev/) "Apache-2.0 WITH LLVM-exception". However, on https://llvm.org/docs/DeveloperPolicy.html#copyright-license-and-patents, we use lower-case "with", so "Apache-2.0 with LLVM-exception". See https://reviews.llvm.org/D96482 for rationale for doing so.
I'd suggest also using "Apache-2.0 with LLVM-exception" here.

Reply:
Hi @kbeyls - thanks for your comment. Do you think we should keep the line "an OSI-approved license" that links to opensource.org considering it's not the exact text?

Reply:
A bit of a side tangent - but maybe we should make sure to check-in the new license in the source repo as well? It was not easy to find the current license text except for that devpolicy page.

Reply:
> Hi @kbeyls - thanks for your comment. Do you think we should keep the line "an OSI-approved license" that links to opensource.org considering it's not the exact text?

I'm not sure if the line "an OSI-approved license" adds much TBH. But I also don't think it's incorrect to have the link to Apache-2.0 as an OSI-approved license. I'm happy both with keeping that reference or removing it.

Reply:
> A bit of a side tangent - but maybe we should make sure to check-in the new license in the source repo as well? It was not easy to find the current license text except for that devpolicy page.

Isn't the canonical location for the license text in the source repo https://github.com/llvm/llvm-project/blob/main/llvm/LICENSE.TXT? Do you propose to move that up one level so it's in the root level in the mono-repo? Or did I misunderstand what you're suggesting above?
Having a LICENSE.TXT file in the root level of the mono-repo makes sense to me, given that is probably the default location where people look first for license text in open source projects - not sure what others would think.

Reply:
Ah yeah I didn't even notice that it was in the `llvm` subdirectory. I can prep a path to just add it to the root as well.

I'll also update the text with your comments above.

Reply:
> Ah yeah I didn't even notice that it was in the `llvm` subdirectory. I can prep a path to just add it to the root as well.
I see that other projects on github (e.g. this one https://github.com/llsoftsec/llsoftsecbook/) have a "View License" on the right hand side. Maybe that's the most canonical way to find a project's license for projects hosted on github. IIRC, the admin of a github project needs to manually set that link. @tstellar - assuming you're an admin for the github LLVM project - do you think it would be useful to set that for the llvm/llvm-project github project? (As an aside, I think we should also set the "Code of Conduct" link there).

Reply:
@kbeyls it actually gets indexed and shows up in the sidebar if you have LICENSE.txt in the root.